src/routes/cart/+page.server.ts

```diff
@@ -9,7 +9,7 @@ import {
 import type { CountryCode, CountryCallingCode } from 'libphonenumber-js';
 import { stripe } from '$lib/stripe-api';
 import type { Stripe } from 'stripe';
-import { env } from '$env/dynamic/private';
+import { BASE_URL } from '$env/static/private';
```

Comment:
@bcaller, these are all notable changes as the values will be included in the build, but no accessible to the client. https://kit.svelte.dev/docs/modules#$env-static-private

There are multiple changes like this throughout the PR due to Svelte 2.x changing the behavior.